coffeecard/CoffeeCard.WebApi/Controllers/v2/AccountController.cs

```diff
@@ -46,5 +48,26 @@ public async Task<ActionResult> Delete()
             await _accountService.RequestAnonymization(user);
             return NoContent();
         }
+        
+        /// <summary>
+        /// Check if a given email is in use
+        /// </summary>
+        /// <param name="request">The email that should be checked</param>
+        /// <returns>Account information</returns>
+        /// <response code="200">Successful request</response>
+        /// <response code="401">Invalid credentials</response>
+        [HttpPost]
+        [ProducesResponseType(typeof(EmailInUseResponse), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(void), StatusCodes.Status401Unauthorized)]
+        [AllowAnonymous]
+        [Route("email-exists")]
+        public async Task<ActionResult<EmailInUseResponse>> EmailInUse([FromBody] EmailInUseRequest request)
```

Comment:
With the update route value, the method name should be updated to align. Same goes with request and response class name 😉